xds/src/test/java/io/grpc/xds/XdsServerBuilderTest.java

```diff
@@ -14,17 +14,23 @@
  * limitations under the License.
  */
 
-package io.grpc.xds.internal.sds;
```

Comment:
if you keep it in the io.grpc.xds.internal.sds, the ServerSdsProtocolNegotiator  doesn't need to expose it as public. also, the Builder is anyways in the sds package it seems better place to be.

Reply:
Yes and I would have liked that. But `XdsClient`, `XdsClientImpl`, and `XdsClient.shutdown` are package private which causes problem for this test unless I make them all public. So making those things public just for a test in SDS didn't sound appealing to me.

Reply:
No change based on my comment.